core/src/com/mygdx/game/client/component/FieldComponent.java

```diff
@@ -0,0 +1,19 @@
+package com.mygdx.game.client.component;
+
+import com.badlogic.ashley.core.Component;
+import com.badlogic.ashley.core.Entity;
+import com.mygdx.game.client.model.Coordinates;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+@Data
+@NoArgsConstructor
+public class FieldComponent implements Component {
```

Comment:
Może zamiast oznaczać to jako "Field" mógłby to był "SlotComponent" albo "ContainerComponent", żeby zwrócić uwagę, że on ma przechowywać inne Entity? I usunąłbym Coordinates tutaj, bo na razie nic chyba nie robią.